services/web/server/src/simcore_service_webserver/studies_dispatcher/_constants.py

```diff
@@ -13,9 +13,9 @@
 
 # This error happens when the linked study ID does not exists OR is not shared with everyone OR is NOT public
 MSG_PUBLIC_PROJECT_NOT_PUBLISHED: Final[str] = (
-    "You need to be logged in to access study with ID '{project_id}'.\n"
-    "Please login and try again.\n"
-    "If you don't have an account, please email to support and request one\n"
+    "You need to be logged in to access this study.<br/><br/>"
+    "Please login and try again.<br/><br/>"
+    "If you don't have an account, please request one at support@osparc.io<br/><br/>"
```

Comment:
Dont hard-code emails since there might be differen tproducts. Use instead ``{support_email}``